libraries/resource_providers/api200/ethernet_network_provider.rb

```diff
@@ -9,10 +9,14 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
+require_relative '../api200/connection_template_provider'
```

Comment:
Unnecessary '../api200/'

Reply:
I needed put it, when I executed the tests I received a error without using that "require_relative".

Reply:
What kind of error? It does not makes any sense, you are still inside the same folder and namespace

Reply:
I tested again after your question and it worked well. =)
I don't know why didn't work good before.

Reply:
Now I noticed that you get it wrong, the what is unnecessary is the `../api200/` i.e. : 
```ruby
require_relative 'connection_template'
```

Reply:
good, but works good without this "require_relative". =)